lti_consumer/tests/unit/test_lti_consumer.py

```diff
@@ -46,6 +46,34 @@ def test_descriptor(self):
         """
         self.assertEqual(self.xblock.descriptor, self.xblock)
 
+    def test_workbench_scenarios(self):
+        """
+        Basic tests that `workbench_scenarios()` returns a well formed scenario.
+        """
+        scenarios = self.xblock.workbench_scenarios()
+        assert isinstance(scenarios, list)
```

Comment:
Again, please replace this with `self.assertIsInstance()`. Using the TestCase methods will often give you more helpful failure messages with test runners, so you get something like "expected list, but scenarios was tuple" instead of a more generic False.

Reply:
Thanks @ormsbee. I wouldn't mind changing this to `self.assertIsInstance()` for consistency with the other tests. But this project uses `pytest` and it offers a far superior debugging messages and clear analysis of what actually happens when native `assert` statements are used.

Unlike `nosetests` which is relatively naive, and can't debug `assert` statements, therefore only `self.assert...()` methods are useful.

What do you think? Should we give up the extra debugging `assert` provides for consistency? or we should keep using `assert` statements and sacrifice some consistency?